apps/back-office/src/app/(dashboard)/appointmentNote/[id]/components/SurveyContentDisplay.tsx

```
@@ -0,0 +1,124 @@
+  const renderValue = () => {
+    if (typeof value === "object") {
+      return (
+        <pre className="whitespace-pre-wrap text-xs">
+          {JSON.stringify(value, null, 2)}
+        </pre>
+      );
+    }
+
+    const stringValue = String(value);
+    if (containsHtmlTags(stringValue)) {
+      return <div dangerouslySetInnerHTML={{ __html: stringValue }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)